.circleci/config.yml

```diff
@@ -10,29 +10,55 @@ jobs:
       - checkout
 
       - run: docker system prune --force
+
       - run:
-          name: Build and Run Docker
+          name: Confirm directory state
+          command: |
+            set -x
+            pwd && ls
+            echo "ls app/my_app"
+            ls app/my_app
+
+      - run:
+          name: Build Docker container
           command: |
             set -x
             docker-compose build
-            docker-compose up -d
             docker-compose run --rm app sh -c "mix --version"
 
-      # - run:
-      #     name: Run Docker container
-      #     command: |
-      #       set -x
-      #       docker-compose up -d
-      #       mix test
-
-      # - run:
-      #     name: Mix test
-      #     command: |
-      #       docker-compose exec app bash -c "cd my_app && mix test"
-
-      # - run:
-      #     name: mix
-      #     command: docker-compose exec app mix archive
+      - run:
+          name: Mix deps.get
+          command: |
+            set -x
+            docker-compose run --rm app sh -c "pwd && ls && cd my_app && mix deps.clean --all && mix deps.get"
+
+      - run:
+          name: Run Docker container
+          command: |
+            set -x
+            docker-compose up -d
+            sleep 10
+            docker ps -f status=running
+
+      - run:
+          name: Confirm exist mix archive
+          command: |
+            set -x
+            docker-compose run --rm app sh -c "pwd && ls && cd my_app && mix archive"
+
+      - run:
+          name: Mix test
+          command: |
+            set -x
+            docker-compose run --rm app sh -c "pwd && ls && cd my_app && mix test"
+
+      - run:
+          name: Mix phx.server run
+          command: |
+            set -x
+            docker-compose run --rm app sh -c "pwd && ls && cd my_app && mix phx.server"
```

Comment:
error

```bash
+ docker-compose run --rm app sh -c 'pwd && ls && cd my_app && mix phx.server'
^@^@/usr/src/app
Dockerfile  my_app
[info] Running MyAppWeb.Endpoint with cowboy 2.8.0 at 0.0.0.0:4000 (http)
[error] Could not start node watcher because script "/usr/src/app/my_app/assets/node_modules/webpack/bin/webpack.js" does not exist. Your Phoenix application is still running, however assets won't be compiled. You may fix this by running "npm install" inside the "assets" directory.
[info] Access MyAppWeb.Endpoint at http://localhost:4000
```